pkg/collector/corechecks/systemd/systemd.go

```diff
@@ -454,6 +483,15 @@ func (c *SystemdCheck) isMonitored(unitName string) bool {
 	return false
 }
 
+func isValidServiceCheckStatus(serviceCheckStatus string) bool {
+	for _, validStatus := range validServiceCheckStatus {
+		if strings.EqualFold(serviceCheckStatus, validStatus) {
```

Comment:
(minor) I'm not sure there is a value of being case insensitive. It's somewhat more confusing IMHO.

Note: Not sure `EqualFold` behave exactly like `ToLower` used above `strings.ToLower(mappedServiceCheckStatus)`